### PR TITLE
Match func_8024140C_EA9EEC

### PR DESCRIPTION
**Pull request by savaughn**
[https://decomp.me/scratch/TG7x3](https://decomp.me/scratch/TG7x3)

### CODE REVIEW

File: src/EA8AE0.c [hunk 1]
```diff
@@ -37,7 +39,13 @@ ApiStatus func_802413FC_EA9EDC(Evt* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "EA8AE0", func_8024140C_EA9EEC);
+ApiStatus func_8024140C_EA9EEC(Evt* script) {
```
Comment (nanaian) [inline src/EA8AE0.c:42]:
```suggestion
ApiStatus func_8024140C_EA9EEC(Evt* script, s32 isInitialCall) {
```

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.00%) on `us`.

⚠️ This PR introduces 2 warnings:
- src/EA8AE0.c:15: warning: function declaration isn't a prototype
- src/EA8AE0.c:45: warning: passing arg 2 of `func_8012DFE8' from incompatible pointer type

File: src/EA8AE0.c (hunk 1 above)
Comment (savaughn) [reply src/EA8AE0.c]: done

https://github.com/pmret/papermario/pull/775/commits/38183feaf1682fe223e9ebe0bc22ad8e06d3c70f

**Comment by nanaian** [comment]
[Removing the & on decomp.me produces no change in output](https://decomp.me/scratch/218t8)

**Comment by savaughn** [comment]
> [Removing the & on decomp.me produces no change in output](https://decomp.me/scratch/218t8)

ok, so decomp.me is the default source of truth?

**Review by nanaian — APPROVED** [review]

**Pull request merged**